modules/labs-react/drawer/lib/DrawerHeader.tsx

```diff
@@ -12,7 +12,7 @@ export interface DrawerHeaderProps extends React.HTMLAttributes<HTMLDivElement>
   /**
    * The function called when the DrawerHeader close button is clicked.
    */
-  onClose?: IconButtonProps['onClick'];
+  onClose?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
Button props no longer extend from a concrete interface since `createComponent` returns an `ElementComponent<'button', ButtonProps>` type that defined Props only in usage. The presence of an `as` will change this interface which is why `ButtonProps` contains no element interface specific to any one element. Uses of `Button` will have to manually determine interfaces used if required. Usage of `mergeProps` and Compound Component APIs in general reduce the need for this though.

The `DrawHeader` component is an example of a complex component that doesn't use the Compound Component API and thus does not give direct access to elements and therefore has a fractured component interface directly creating "prop holes" to access underlying elements. This will still happen in application components where Canvas Kit components are wrapped in configuration APIs. I don't have a direct way to codemod this though...